course02/homework00/skeleton/vector.cc

```diff
@@ -0,0 +1,26 @@
+#include "vector.h"
+
+int VectorInt::capacity() const
+{
+	return mMaxSize;
+}
+
+int VectorInt::size() const
+{
+	return mSize;
+}
+
+int VectorInt::back() const
+{
+	return mData[mSize -1];
+}
+
+void VectorInt::push_back(const int& x)
+{
+	mData[mSize++] = x;
```

Comment:
same remark here on assert

Reply:
not sure what you mean here,  getting the back of an empty vector should be caught, but you can always push_back

Reply:
oh i guess you mean that i'm not hitting capacity